pyuvsim/version.py

```diff
@@ -54,7 +54,9 @@ def construct_version_info():
 
     try:
         git_origin = _get_git_output(['config', '--get', 'remote.origin.url'], capture_stderr=True)
-        if git_origin.split('/')[-1] != 'pyuvsim.git':  # pragma: no cover
+        print('git origin:', git_origin)
```

Comment:
I thought we updated to using `pygitversion` in pyuvsim? Or has that not been merged yet?

Reply:
We don't, yet, nor is there an outstanding PR to do so. I actually started to change it in this PR and then decided to wait for a different for a new one to limit the number of things I changed in this PR.

Reply:
Great. Maybe I'm thinking of `pyuvdata` haha. I get so confused.